test/cases/core/payments/routing.test.ts

```diff
@@ -0,0 +1,136 @@
+
+describe('payments routing', () => {
+    let nodes: LuminoNodeList;
+    let env: LuminoTestEnvironment;
+
+    // starting balance for each node
+    const aliceDaveLumDeposit = toWei(
+        mediated.channels[7].participant2.deposit
+    );
+    const daveAliceLumDeposit = toWei(
+        mediated.channels[7].participant1.deposit
+    );
+    const daveEveLumDeposit = toWei(mediated.channels[8].participant1.deposit);
+    const eveLumDeposit = toWei(mediated.channels[8].participant2.deposit);
+
+    // starting balance for each node
+    const aliceBobRifDeposit = toWei(mediated.channels[0].participant2.deposit);
+    // starting balance for each node
+    const aliceDaveRifDeposit = toWei(
+        mediated.channels[3].participant2.deposit
+    );
+
+    const paymentAmount = toWei(1);
+
+    beforeAll(async () => {
+        env = await setupTestEnvironment(mediated);
+        nodes = env.nodes;
+    }, Timeouts.SETUP);
+
+    afterAll(async () => {
+        await env.stop();
+    }, Timeouts.TEARDOWN);
+
+    it(
+        'should pay through shortest path',
+        async () => {
+            // shortest Path is alice -> dave -> eve
+            const payment: PaymentParams = {
+                tokenAddress: tokenAddresses.LUM,
+                partnerAddress: nodes.eve.client.address,
+                amountOnWei: paymentAmount
+            };
+
+            await nodes.alice.client.sdk.makePayment(payment);
+
+            // verify alice -> eve
+            await given(nodes.alice)
+                .expectChannel({
+                    tokenAddress: tokenAddresses.LUM,
+                    partnerAddress: nodes.dave.client.address
+                })
+                .toHaveBalance(aliceDaveLumDeposit - paymentAmount);
+
+            await given(nodes.dave)
+                .expectChannel({
+                    tokenAddress: tokenAddresses.LUM,
+                    partnerAddress: nodes.eve.client.address
+                })
+                .toHaveBalance(daveEveLumDeposit - paymentAmount);
+
+            // now verify charlie -> alice
+            await given(nodes.eve)
+                .expectChannel({
+                    tokenAddress: tokenAddresses.LUM,
+                    partnerAddress: nodes.dave.client.address
+                })
+                .toHaveBalance(eveLumDeposit + paymentAmount);
+
+            await given(nodes.dave)
+                .expectChannel({
+                    tokenAddress: tokenAddresses.LUM,
+                    partnerAddress: nodes.alice.client.address
+                })
+                .toHaveBalance(daveAliceLumDeposit + paymentAmount);
+        },
+        Timeouts.TEST
+    );
+
+    it(
+        'should notify error if no path',
+        async () => {
+            const payment: PaymentParams = {
+                tokenAddress: tokenAddresses.RIF,
+                partnerAddress: nodes.eve.client.address,
+                amountOnWei: toWei(0.00001)
```

Comment:
I used low amount due to low balance of RIF in the account used to fund. I tried to add more balance to the accounts, but I couldn't.

Reply:
RIF token is a limited amount token, we have 50 tokens on the origin account